integration-tests/benchmark/benchmark_runner_test.go

```diff
@@ -0,0 +1,370 @@
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+)
+
+func init() {
+	logging.Init()
+}
+
+var baseEnvironmentConfig = &environment.Config{
+	TTL: time.Hour * 720, // 30 days,
+}
+
+var dynamicValues_EvmNodes = []map[string]interface{}{
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_0"),
+	},
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_1"),
+	},
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_2"),
+	},
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_3"),
+	},
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_4"),
+	},
+	{
+		"EVM_NODES": os.Getenv("EVM_NODES_5"),
+	},
+}
+
+var chainlinkPerformance = map[string]interface{}{
+	"chainlink": map[string]interface{}{
+		"resources": map[string]interface{}{
+			"requests": map[string]interface{}{
+				"cpu":    "1000m",
+				"memory": "4Gi",
+			},
+			"limits": map[string]interface{}{
+				"cpu":    "1000m",
+				"memory": "4Gi",
+			},
+		},
+	},
+	"db": map[string]interface{}{
+		"resources": map[string]interface{}{
+			"requests": map[string]interface{}{
+				"cpu":    "1000m",
+				"memory": "1Gi",
+			},
+			"limits": map[string]interface{}{
+				"cpu":    "1000m",
+				"memory": "1Gi",
+			},
+		},
+		"stateful": true,
+		"capacity": "20Gi",
+	},
+}
+
+var chainlinkSoak = map[string]interface{}{
+	"chainlink": map[string]interface{}{
+		"resources": map[string]interface{}{
+			"requests": map[string]interface{}{
+				"cpu":    "350m",
+				"memory": "1Gi",
+			},
+			"limits": map[string]interface{}{
+				"cpu":    "350m",
+				"memory": "1Gi",
+			},
+		},
+	},
+	"db": map[string]interface{}{
+		"resources": map[string]interface{}{
+			"requests": map[string]interface{}{
+				"cpu":    "250m",
+				"memory": "256Mi",
+			},
+			"limits": map[string]interface{}{
+				"cpu":    "250m",
+				"memory": "256Mi",
+			},
+		},
+		"stateful": true,
+		"capacity": "20Gi",
+	},
+}
+
+func TestKeeperBenchmarkSimulated(t *testing.T) {
+	KeeperBenchmarkSimulated(t, "@simulated-registry-1-3 @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkSimulated1_2(t *testing.T) {
+	KeeperBenchmarkSimulated(t, "@simulated-registry1-2 @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkSimulatedMultiple(t *testing.T) {
+	KeeperBenchmarkSimulated(t, "@simulated-multiple-registries @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkGoerli(t *testing.T) {
+	KeeperBenchmarkGoerli(t, "@goerli-registry-1-3 @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkGoerli1_2(t *testing.T) {
+	KeeperBenchmarkGoerli(t, "@goerli-registry1-2 @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkGoerliMultiple(t *testing.T) {
+	KeeperBenchmarkGoerli(t, "@goerli-multiple-registries @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkArbitrumGoerli(t *testing.T) {
+	KeeperBenchmarkArbitrumGoerli(t, "@arbitrum-goerli-registry-1-3 @benchmark-keeper")
+}
+
+func TestKeeperBenchmarkOptimisticGoerli(t *testing.T) {
+	KeeperBenchmarkOptimisticGoerli(t, "@optimistic-goerli-registry-1-3 @benchmark-keeper")
+}
```

Comment:
nit. these functions might benefit from refactoring to make clear the differences among them

Reply:
Fixed this in https://github.com/smartcontractkit/chainlink/pull/7747